students.txt

```diff
@@ -14,7 +14,13 @@ List of students
 7. Daniil I.
 8. Daria P. 
 9. Max R.
-10. Dima Ch.
+10. Dima Chalenko
+Favorite activities
+  .reading books (psychology, proramming)
+  .program
+  .like to sleep a lot
+  .love play gaming
+  p.s. I want to start working on my profile as soon as possible :)
```

Comment:
You forgot to add one clear line after the last line of your description. Plus, please move this line one space to the left, so that it doesn't seem like an item in the list.